pom.xml

```diff
@@ -54,11 +54,10 @@
 
   <dependencies>
 
-    <!-- Probably works with 3.0 as well -->
     <dependency>
-      <groupId>javax.servlet</groupId>
-      <artifactId>javax.servlet-api</artifactId>
-      <version>3.1.0</version>
+      <groupId>jakarta.servlet</groupId>
+      <artifactId>jakarta.servlet-api</artifactId>
+      <version>5.0.0</version>
```

Comment:
I wonder why you used 5 instead of 6.0.0 which is latest?

Reply:
For maximum compatibility. Some webapp containers only support `5.0.0`.

Reply:
I agree that it should stay on 5.0 for JakartaEE 9 compatibility as e.g. used by Jetty 11. Any use in a JakartaEE 10 / Servlet 6 container should be backwards-compatible to that.